src/Lykke.Service.FeeCalculator/Controllers/CashoutFeesController.cs

```diff
@@ -20,21 +20,30 @@ public CashoutFeesController(IDummySettingsHolder dummySettingsHolder)
         {
             _dummySettingsHolder = dummySettingsHolder;
         }
-        
+
         [HttpGet]
         [SwaggerOperation("GetCashoutFees")]
-        [ProducesResponseType(typeof(List<CashoutFee>), (int)HttpStatusCode.OK)]
-        [ProducesResponseType(typeof(ErrorResponse), (int)HttpStatusCode.InternalServerError)]
+        [ProducesResponseType(typeof(List<CashoutFee>), (int) HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(ErrorResponse), (int) HttpStatusCode.InternalServerError)]
         public IActionResult GetCashoutFees([FromQuery] string assetId = null)
         {
             if (string.IsNullOrWhiteSpace(assetId))
                 return Ok(_dummySettingsHolder.GetCashoutFees());
 
-            assetId = assetId.ToUpper();
-            var fees = _dummySettingsHolder.GetCashoutFees().Where(fee => fee.AssetId == assetId).ToList();
+            var assetIdUpper = assetId.ToUpper();
+            var fees = _dummySettingsHolder.GetCashoutFees().Where(fee => fee.AssetId.ToUpper() == assetIdUpper)
```

Comment:
fee.AssetId.Equals(assetId, StringComparison.InvariantCultureIgnoreCase))
and remove previous line.

Reply:
You are right )